### PR TITLE
MODELS page

### PR DESCRIPTION
**Pull request by clmarchesan**
Hi Louis, Frédéric,

I don't know if the MODELS folder hierarchy is fine,
if you agree, we can leave it as is. My changes are very small and I can redo them in another location.

Claudio

### CODE REVIEW

**Comment by ftessier** [comment]
@marmetl let me clean up this branch before you merge it in. I'll wait for the review or conversation to be resolved, then I can prune it.

**Comment by clmarchesan** [comment]
If you can, please delete all but the last commits for me. I was trying to understand how it works.

I prefer to keep the pdf because it's 111 pages with Word formulas. Also, people are used to downloading pdfs.

Regards,
Claudio

**Comment by clmarchesan** [comment]
Hi @ftessier,

I wrote the readme.md without thinking of search engines. I like it because it is synthetic for the reader but perhaps it is not optimized to be indexed.

I will also see what others will do before I change.

**Comment by marmetl** [comment]
> I don't know if the MODELS folder hierarchy is fine, if you agree, we can leave it as is.

The original website has not been transferred yet, the folders may change in the process.  Just having a MODELS folder with everybody's files in it will just lead to an unmanageable mess.  One possibility is to have subfolders, one for each author, but this is to be decided later.

**Comment by marmetl** [comment]
>  My changes are very small and I can redo them in another location.

The changes are **very large**.  The file "*Using the Doppler type redshift in Galactic Recession*" was uploated five times, that's 5 $\times$ 778 KB, and "*Key features of the 4-Sphere cosmological model*" was uploaded five times as well, that's 5 $\times$ 190 KB, *for a total of 4.8 MB already used in the repository's history*.  (GitHub may not be able to recognize that you only changed a few words in a .pdf file.)

There is a limit of 2 GB for the ACG organization.  At this rate, and if other members upload .pdf papers as you do, we will be near the memory limit in only a year!

The *squash and merge* suggested by @ftessier should reduce the amount of required memory to only 968 KB, so I will also encourage you to add your paper in markdown format to reduce the number of .pdf versions stored in memory.

We should wait for more comments (on the scientific content) from other members before merging this branch.

**Comment by ftessier** [comment]
Indeed, when squashing etc. the multiple versions of the binary `.pdf` objects will be pruned (maybe not immediately), leaving only the one that was committed last.

**Comment by clmarchesan** [comment]
would deleting the fork solve it? you can cancel whenever you want.

**Comment by clmarchesan** [comment]
I would delete the fork (I have the files on my computer) but I don't know how. Space could become a problem, then we decide what to do.

**Comment by clmarchesan** [comment]
Ok perfect @ftessier, thanks

**Comment by marmetl** [comment]
@ftessier if you can 'squash and merge' the file README.md but not the .pdf file, that would be great.
That should fit in with the new folder structure, which contains `models` at the root level of the repo.  Thanks!

File: models/4seu/The 4-Sphere model for an Alternative Metric in Cosmology - Key Aspects and Speculation.pdf
Comment (marmetl) [inline models/4seu/The 4-Sphere model for an Alternative Metric in Cosmology - Key Aspects and Speculation.pdf:1]: If possible, delete this .pdf file.  We can add it later to another folder.

**Comment by clmarchesan** [comment]
Ok, I deleted it

**Comment by marmetl** [comment]
@clmarchesan your page is now displayed [on the website](https://a-cosmology-group.github.io/acg/models/4seu/)
What do you think?

Your folder hierarchy works well.  You can edit the webpage on the [`a-cosmology-group` repo](https://github.com/a-cosmology-group/acg/blob/main/models/4seu/index.md), just make sure you don't change the *YAML front matter* at the beginning of the page.  This is required for the page to be displayed on the website.

New versions of your documents will all go in the [`models/4seu/`](https://github.com/marmetl/acg/tree/main/models/4seu) folder.  You can update your forked version of the `a-cosmology-group/acg` repo. I would recommend to use a text file (e.g. `paper2-on-cosmology.md`) which could be converted to .pdf once it is ready.  

Your `clmarchesan:main` fork can be merged to `a-cosmology-group:main` from time to time, e.g. when you have made enough changes that warrant discussions.

**Comment by clmarchesan** [comment]
Hi Louis,

The page turned out well even if the link points to an old document. The
new updated PDF (that we just deleted) was supposed to replace it. Let me
know when the replacement will be possible.

It's okay to limit updates to large documents (When I built the fork I
thought you could delete what you wanted).

Claudio

**Comment by marmetl** [comment]
Sorry about deleting your most recent .pdf version, I am learning how to make the website and wanted to make only one small change at a time.

Since it works, replacement is possible now!  Sync your fork, delete the old .pdf, add the new one and open a pull request, I'll approve it right away so the most recent version is on the webpage.

**Comment by clmarchesan** [comment]
Hi Louis, I don't want to be wrong again.

Sync asks me:

This branch is out-of-date

1 - Update branch to merge the latest changes from the upstream repository
into this branch.

2 - Discard 46 commits to make this branch match the upstream repository.
46 commits will be removed from this branch


I would answer to discard the 46 commits.

Then I would delete 4seu.pdf, commit and then upload the new pdf, naming it
as 4seu.pdf, and commit again. Is this correct?

Step 1 - delete
             commit
Step 2 - uploads
             commit

Claudio

**Comment by marmetl** [comment]
Yes to all of the above!

When you upload the new `.pdf`, you can either
- rename it as `4seu.pdf`,
or
- keep the `.pdf` name that is convenient for you and correct the link in `index.md`

(P.S. You were not wrong, you were ahead of an existing structure for the website!)

**Comment by clmarchesan** [comment]
Hi, I got 3 errors similar to these I don't know if you've seen them

https://mail.google.com/mail/u/0/?tab=rm&ogbl#inbox/FMfcgzGsnBmKQvRPkRdPZFnjRFHrpSfr

Claudio

**Comment by marmetl** [comment]
Hi, the link wants me to create a `@gmail.com` account (which I won't do!)

Can you copy the text of these *error messages* here?

**Comment by clmarchesan** [comment]
[clmarchesan/acg] Run failed: Deploy Jekyll site to Pages - main (c54edfd)
Inbox
Claudio Marchesan ***@***.***>
Sat, Jun 17, 6:49 AM (23 hours ago)
to clmarchesan/acg, Ci

[image: GitHub][clmarchesan/acg] Deploy Jekyll site to Pages workflow run

 Deploy Jekyll site to Pages: Some jobs were not successful

View workflow run
<https://github.com/clmarchesan/acg/actions/runs/5296397472>

[image: build]

Deploy Jekyll site to Pages / build
Failed in 34 seconds
[image: annotations for Deploy Jekyll site to Pages / build] 2
<https://github.com/clmarchesan/acg/actions/runs/5296397472>
[image: deploy]

Deploy Jekyll site to Pages / deploy
Skipped



—
You are receiving this because you are subscribed to this thread.
Manage your GitHub Actions notifications
<https://github.com/settings/notifications>



GitHub, Inc. ・88 Colin P Kelly Jr Street ・San Francisco, CA 94107


Annotations
2 errors
*build*
<https://github.com/clmarchesan/acg/actions/runs/5296397472/jobs/9587479046#step:4:6>
Get Pages site failed. Please verify that the repository has Pages enabled
and configured to build using GitHub Actions, or consider exploring the
`enablement` parameter for this action.
*build*
<https://github.com/clmarchesan/acg/actions/runs/5296397472/jobs/9587479046#step:4:7>
HttpError: Not Found
ReplyReply allForward

**Comment by marmetl** [comment]
@clmarchesan these are **not your** errors.
Every time you *push* changes on your `main`-branch, GitHub uses `Actions` to build the entire website from your repo `clmarchesan/acg/`.
However, your account is not setup for that so error messages are generated.

If you don't plan to use `Actions` you can disable this behaviour:
- click [`clmarchesan/acg`](https://github.com/clmarchesan/acg),
-  --> *top-menu* ` ⚙️ Settings`,
-  --> *left-menu* ` ▶️ Actions` --> `General` and,
- under *Actions permissions* click `Disable actions` and `Save`.

Let me know it this works.

**Comment by clmarchesan** [comment]
hI, It worked.

Claudio

**Comment by marmetl** [comment]
@clmarchesan I moved your webpage to the [`time-dependent-cosmologies`](https://github.com/a-cosmology-group/acg/tree/main/models/time-dependent-cosmologies) folder so it would appear in the list with other models on the [ACG webpage](https://a-cosmology-group.github.io/acg/models/time-dependent-cosmologies/)

You will have to sync your fork and --> `Update branch` before editing the webpage.

Hopefully this is the last time I change my mind about the folder structure of the website 😄 

sync-fork--update-branch

**Comment by clmarchesan** [comment]
Hi @marmetl thanks for moving it.

**Pull request merged**